src/mame/layout/connect4.lay

```diff
@@ -640,77 +720,90 @@ license:CC0
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd10" ref="vfd0" state="0">
-			<bounds x="45" y="120" width="9" height="14"/>
+		<element name="lamp115" ref="ledseg1" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd9" ref="vfd0" state="0">
-			<bounds x="54" y="120" width="9" height="14"/>
+		<element name="lamp116" ref="ledseg2" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd8" ref="vfd0" state="0">
-			<bounds x="63" y="120" width="9" height="14"/>
+		<element name="lamp117" ref="ledseg3" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd7" ref="vfd0" state="0">
-			<bounds x="72" y="120" width="9" height="14"/>
+		<element name="lamp112" ref="ledseg4" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd6" ref="vfd0" state="0">
-			<bounds x="81" y="120" width="9" height="14"/>
+		<element name="lamp113" ref="ledseg5" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd5" ref="vfd0" state="0">
-			<bounds x="90" y="120" width="9" height="14"/>
+		<element name="lamp118" ref="ledseg6" state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+			<bounds x="9" y="56" width="8" height="10"/>
 		</element>
-		<element name="vfd4" ref="vfd0" state="0">
-			<bounds x="99" y="120" width="9" height="14"/>
+
+		<element name="lamp119" ref="timeout" state="0">
+			<bounds x="30" y="56" width="28" height="14"/>
 		</element>
-		<element name="vfd3" ref="vfd0" state="0">
-			<bounds x="108" y="120" width="9" height="14"/>
+
+		<element name="lamp122" ref="play" state="0">
+			<bounds x="0" y="140" width="28" height="14"/>
 		</element>
-		<element name="vfd2" ref="vfd0" state="0">
-			<bounds x="117" y="120" width="9" height="14"/>
+		<element name="lamp121" ref="pass" state="0">
+			<bounds x="40" y="140" width="28" height="14"/>
 		</element>
-		<element name="vfd1" ref="vfd0" state="0">
-			<bounds x="126" y="120" width="9" height="14"/>
+		<element name="lamp123" ref="select" state="0">
+			<bounds x="0" y="90" width="28" height="14"/>
 		</element>
-		<element name="vfd0" ref="vfd0" state="0">
-			<bounds x="135" y="120" width="9" height="14"/>
+		<element name="lamp120" ref="drop" state="0">
+			<bounds x="30" y="90" width="28" height="14"/>
 		</element>
-	</view>
-</mamelayout>
 
 
+		<repeat count="16">
+			<param name="i" start="15" increment="-1"/>
+			<param name="x" start="0" increment="9"/>
+			<element name="vfd~i~" ref="vfd0">
+				<bounds x="~x~" y="120" width="9" height="14"/>
+			</element>
+		</repeat>
+
+	</view>
+</mamelayout>
```

Comment:
You’re missing the final newline on this file.

Reply:
Turns out my autorun of srcclean doesn't work any more, I'll resolve this with the rest